src/components/navigation-bar/ui/auth/AuthBalance.tsx

```diff
@@ -0,0 +1,49 @@
+import { FC } from "react";
+import { useStyletron } from "baseui";
+import { PRIMITIVE_COLORS } from "../../../../shared";
+import { LabelSmall } from "baseui/typography";
+import { expandProperty } from "inline-style-expand-shorthand";
+
+export type AuthBalanceProps = {
+  amount: string;
+  coin?: string;
```

Comment:
should be renamed to cents. coins is too misleading with currency. And mean literally coin.